ironic_client/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM docker.io/centos:centos7
+
+RUN yum install -y python-requests
+RUN curl https://raw.githubusercontent.com/openstack/tripleo-repos/master/tripleo_repos/main.py | python - current-tripleo
+RUN yum install -y python-ironicclient python-openstackclient jq
+
+RUN echo "#!/usr/bin/bash" > /bin/runironic_client
+RUN echo "set -ex" >> /bin/runironic_client
+RUN echo "export OS_TOKEN=fake-token" >> /bin/runironic_client
+RUN echo "export OS_URL=http://localhost:6385/" >> /bin/runironic_client
+RUN echo "openstack baremetal create /mnt/master_nodes.json" >> /bin/runironic_client
+RUN echo "mkdir -p configdrive/openstack/latest" >> /bin/runironic_client
+RUN echo "cp /mnt/master.ign configdrive/openstack/latest/user_data" >> /bin/runironic_client
+RUN echo "for node in \$(jq -r .nodes[].name /mnt/master_nodes.json); do" >> /bin/runironic_client
+RUN echo "  openstack baremetal node set \$node  --instance-info image_source=http://172.22.0.1/images/redhat-coreos-maipo-47.284-openstack.qcow2 --instance-info image_checksum=2a38fafe0b9465937955e4d054b8db3a --instance-info root_gb=25 --property root_device='{\"name\": \"/dev/vda\"}'" >> /bin/runironic_client
+RUN echo "  openstack baremetal node manage \$node --wait" >> /bin/runironic_client
+RUN echo "  openstack baremetal node provide \$node --wait" >> /bin/runironic_client
+RUN echo "  openstack baremetal node deploy --config-drive configdrive \$node" >> /bin/runironic_client
```

Comment:
We'll no longer need to mention all of this in the README, and we wont need to install openstack clients their either. Actually wouldn't putting all of this in a container make it a little more difficult to retrace parts of the process while developing? So I guess my question is, why do we need to do this, is it to rely less on the virt host?

Reply:
Yeah we could run all this from the virt-host, I was trying to automate things, and yeah rely less on the virthost (prototype the pattern we expect for the final solution).

This could be premature though, maybe I should just run this script from 05* directoy for the time being?  No strong opinion but I'd like the master control-plane deployment to be automated one way or antother.

Reply:
I feel like it might help us iterate a bit faster if we kept this out of a container for the moment. So my preference would be for a script being called from 05 directly. But either would work so go with what ever your happier with.

Reply:
Ok good point, I'll abandon this for now and move the script to the host, we can always package things in containers later when we're ready, thanks for the feedback